wordpress-mobile/WordPress-iOS
# Blogging Prompts Feature Introduction: show scheduling view

Ref: #18176 

Showing the Blogging Prompts Feature Introduction with actionable buttons is now handled by a presenter, `BloggingPromptsIntroductionPresenter`. It handles:
- showing the FI.
- showing the scheduling view when `Remind me` is tapped.

There are stubs in place right now, but in the future it will handle:
- showing post creation when `Try it now` is tapped.
- showing the site selector if the account has more than one site.

To test:
- Enable the `bloggingPrompts` feature.
- Log into the app with an account that has one site.
- On the Feature Introduction, tap `Remind me`.
  - Note: you may need to minimize/maximize the app after immediately logging in to get the FI to show. I think the Quick Start prompt interrupts it.	
- Verify:
  - The Feature Introduction is dismissed.
  - The scheduling flow is shown. 
    - Depending on if you've viewed blogging reminders before, you may(not) see the Reminders Intro (`Set your blogging reminders`) sheet before the calendar view. This is controlled by the Blogging Reminders flow.
- Log into the app with an account that has multiple sites.
  - On the FI, verify tapping both buttons simply dismisses the view (for now).

| FI | Reminders Intro | Schedule |
|--------|-------|-------|
| ![fi](https://user-images.githubusercontent.com/1816888/164553873-35ef3e70-f401-464c-b195-2e6a705fc1db.png) | ![intro_sheet](https://user-images.githubusercontent.com/1816888/164553888-2f789c88-f85e-4552-9de2-e0f5583eaacf.png) | ![calendar](https://user-images.githubusercontent.com/1816888/164553885-90eef4ef-320f-4283-9ef9-ef0f1e10fde3.png) |


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
